### PR TITLE
Remove content-register

### PR DESCRIPTION
**Pull request by danielroseman**
Once this is merged and puppet has removed everything from the servers, there will be a follow-up PR to remove the puppet classes themselves.

The db will need to be dropped manually.

### CODE REVIEW

**Comment by elliotcm** [comment]
Just missing https://github.com/alphagov/govuk-puppet/blob/remove-content-register/hieradata/staging.yaml#L12

**Comment by danielroseman** [comment]
@elliotcm added.

**Comment by elliotcm** [comment]
👍

File: hieradata/class/backend.yaml [hunk 1]
```diff
@@ -51,7 +51,6 @@ govuk::node::s_base::apps:
   - canary_backend
   - collections_publisher
   - contacts
-  - content_register
```
Comment (alexmuller) [inline hieradata/class/backend.yaml]: I don't think you want to do this just yet - it needs to stay in here for the `ensure => absent` to take effect.

Comment (danielroseman) [inline hieradata/class/backend.yaml]: @alexmuller removed.

**Pull request merged**